Jenkinsfile

```diff
@@ -89,11 +98,16 @@ spec:
 
     parameters {
         choice(name: 'Action',
-                choices: ['STANDARD', 'RELEASE'],
-                description: 'Kind of running : \nSTANDARD : (default) classical CI\nRELEASE : Build release')
-        booleanParam(name: 'FORCE_SONAR', defaultValue: false, description: 'Force Sonar analysis')
-        string(name: 'EXTRA_BUILD_ARGS', defaultValue: "", description: 'Add some extra parameters to maven commands. Applies to all maven calls.')
-        string(name: 'POST_LOGIN_SCRIPT', defaultValue: "", description: 'Execute a shell command after login. Useful for maintenance.')
+           choices: ['STANDARD', 'RELEASE'],
```

Comment:
Could we review here to make it easier to read ?

Reply:
It was to uniform with component runtime, and try to limit the file length,
Those parameters are really rarely edited, and they were taking a lot of place before.
In editor, written like this, you have quickly the name and default value. 
![image](https://user-images.githubusercontent.com/1255625/195616849-1f392a03-21cc-4c80-ba1a-b95222f50100.png)
Any other proposal?